src/azure-cli-core/azure/cli/core/commands/parameters.py

```diff
@@ -96,9 +95,16 @@ def __contains__(self, other):
         return next((True for x in self if other.lower() == x.lower()), False)
 
 
+def model_choice_list(resource_type, model_name):
+    model = get_versioned_models(resource_type, model_name)
+    return enum_choice_list(model) if model else {}
```

Comment:
`enum_choice_list` returns an empty dict so it made sense to do the same here.
Actually since the usage is `**model_choice_list`, it has to be a dict, not a list.
```
$ python
>>> val = {'n': 1}
>>> 
>>> def f(**kwargs):
...     print(kwargs)
... 
>>> f(**val)
{'n': 1}
>>> 
>>> l = []
>>> f(**l)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: f() argument after ** must be a mapping, not list
>>>
```